website/src/app/kb/authenticate/user-group-sync/readme.mdx

```diff
@@ -1,70 +1,77 @@
 
-Firezone supports user / group sync from Google Workspace. This feature is
-**automatically enabled** for the
-[Google Workspace connector](/kb/authenticate/google). Once the connector is
-activated, users, groups, and organizational units will be synced from your
-Google Workspace account every few minutes.
-
-<Alert
-  color="info"
-  html={`
-    User / group sync is only supported with the Google Workspace connector
-    at this time. Support for more providers is currently
-    <a href="https://github.com/firezone/firezone/issues/2016">in development</a>.
-  `}
-/>
+</PlanBadge>
 
-## Deleting entities
+Firezone supports automatic user / group sync from Google Workspace, Microsoft
+Entra ID, and Okta. This feature is **automatically enabled** when you create
+one of the [Google Workspace](/kb/authenticate/google),
+[Microsoft Entra](/kb/authenticate/entra), or [Okta](/kb/authenticate/okta)
+connectors. No further configuration is necessary. Once the connector is
+activated, users, groups, and organizational units will be synced from your
+identity provider every few minutes.
 
-To preserve audit log trails, Firezone <strong>never</strong> deletes synced
-users or groups.
+## Deleting or suspending users
 
-### Deleting or suspending users
+Firezone <strong>never</strong> deletes synced users or groups. Instead, they're
+disabled temporarily in Firezone when they're deleted or suspended in your
+identity provider. This helps to preserve audit trails and other logged activity
+within Firezone.
 
-When a user is deleted or suspended in your Google Workspace account, Firezone
-will disable the user and clear all active client and admin portal web sessions
-for that user upon the next user/group sync. The user will be **signed out of
-all clients** and forced to reauthenticate.
+When a user is deleted or suspended in your identity provider, Firezone will
+disable the user and clear all active client and admin portal web sessions for
+that user upon the next sync. The user will be **signed out of all clients** and
+forced to reauthenticate.
 
 This ensures terminated employees will have all Firezone access revoked within a
-few minutes of deleting or suspending them in your Google Workspace account.
+few minutes of deleting or suspending them in your identity provider.
 
 ### Deleting a group or organizational unit
 
-Deleting a group or organizational unit in Google Workspace will hide the group
-and associated policy in Firezone.
+Deleting a group or organizational unit in your identity provider will hide the
+group and disable the associated policy in Firezone.
 
 ## Nested groups and organizational units
 
-Firezone flattens nested groups and organizational units synced from Google
-Workspace. User membership is determined **only** by its immediate parent. For
-example, if you had the following group structure in your Google Workspace
-account:
+Firezone syncs transitive memberships from your identity provider. This means
```

Comment:
@bmanifold just wanted to confirm if we do it for Okta too? Probably it should be a pre-provider feature description and statement that we try to do it whenever the provider supports fetching transitive memberships.